docs/src/main/paradox/index.md

```diff
@@ -19,7 +23,15 @@ Based on a protobuf service definition, akka-grpc can generate:
 * On the server side, code to create an Akka HTTP route based on your implementation of the API
 * On the client side, a client for the API.
 
+### gRPC vs REST
+
+* Where REST is more flexible about transport and encoding, gRPC standardizes on HTTP/2 and Protobuf.
+* Where REST can be either schemaless or use a 3rd-party schema, gRPC always declares the service and messages in a Protobuf schema definition.
+
+### gRPC vs SOAP
 
+* Where SOAP is more flexible about transport and encoding, gRPC standardizes on HTTP/2 and Protobuf.
+* Where in SOAP protocols are often set in stone once defined, Protobuf is explicitly intended to support schema evolution.
```

Comment:
Second time someone reminds me of SOAP today. Yuck!

"defined, and may require a new path for every version of the service, " ?

Reply:
Heh, should we avoid mentioning it at all? Good addition.

Reply:
I think it makes sense, because I think this is a pro of gRPC compared to SOAP (otoh pretty much anything is a pro compared to SOAP ;) )